doc/changelog.rst

```diff
@@ -24,6 +24,11 @@ Bug fixes
 - Fix front manufacturer modified filter to consider only visible shop products
 - Make Xtheme TextPlugin fallback to empty string when there is no translation for the current language.
 
+Admin
+~~~~~
+
+- Make Summernote editor use Shuup media browser when adding pictures
```

Comment:
Should this mention that the drag and drop has been disabled as well? and why perhaps?